examples/basic_example_v1/basic_example.py

```diff
@@ -4,6 +4,17 @@
 import ydb
 import basic_example_data
 
+# Table path prefix allows to put the working tables into the specific directory
+# inside the YDB database. Putting `PRAGMA TablePathPrefix("some/path")`
+# at the beginning of the query allows to reference the tables through
+# their names "under" the specified directory.
+#
+# TablePathPrefix can be defined as an absolute path (starting with slash),
```

Comment:
This phrase is interesting, does table path prefix supports relative path for YDB?

Reply:
> This phrase is interesting, does table path prefix supports relative path for YDB?

Good point. Seems that I was confused by the docs, in particular with that thing: https://ydb.tech/ru/docs/yql/reference/syntax/pragma#table-path-prefix

Hard to understand why does it need the database part on the first place...